### PR TITLE
chat commands: add !cmb command

### PR DESCRIPTION
**Pull request by MagicfTail**
Add a command showing the combat level and combat related stats similar to osbuddy

image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java [hunk 1]
```diff
@@ -141,6 +148,11 @@ public void onSetMessage(SetMessage setMessage)
 			log.debug("Running total level lookup");
 			executor.submit(() -> playerSkillLookup(setMessage.getType(), setMessage, "total"));
 		}
+		else if (config.lvl() && COMBAT_ABBREVIATIONS.stream().anyMatch(message.toLowerCase()::equals))
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java]: Cant you just use contains(message.toLowercase)?

**Comment by MagicfTail** [comment]
Fixes #4561

File: runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java [hunk 2]
```diff
@@ -65,8 +67,12 @@
 {
 	private static final float HIGH_ALCHEMY_CONSTANT = 0.6f;
 
+	private static final ImmutableList<String> COMBAT_ABBREVIATIONS = ImmutableList.of("!combat", "!cmb", "!combatlevel", "!cmblvl");
+
 	private final HiscoreClient hiscoreClient = new HiscoreClient();
 
+	private HiscoreResult playerStats;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java]: Why is this a field?

File: runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java [hunk 3]
```diff
@@ -65,8 +67,12 @@
 {
 	private static final float HIGH_ALCHEMY_CONSTANT = 0.6f;
 
+	private static final ImmutableList<String> COMBAT_ABBREVIATIONS = ImmutableList.of("!combat", "!cmb", "!combatlevel", "!cmblvl");
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java]: For now just !cmb is enough I guess

File: runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java (hunk 2 above)
Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java]: That's a good question, it no longer is

File: runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java [hunk 4]
```diff
@@ -189,6 +190,11 @@ public void onSetMessage(SetMessage setMessage)
 			log.debug("Running total level lookup");
 			executor.submit(() -> playerSkillLookup(setMessage, "total"));
 		}
+		else if (config.lvl() && message.toLowerCase().equals("!cmb"))
+		{
+			log.debug("Running combat level lookup");
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java]: Doesn't `HighscoreClient.lookup` already log when performing a lookup?

Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java]: It tells you it build an URI, but not why. Every other other command specify what command was run

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java]: Ah, that's fine then I guess. :woman_shrugging:

**Pull request merged**